test/test.cpp

```diff
@@ -25,6 +25,9 @@
 
 #include "theme/theme.h"
 
+// Run thread tests through the address sanitizer
+#include "thread/TestKanoWorldRefresh.h"
```

Comment:
Just a question: If we're making a separated multi-thread tests program, why are we including it in the unit ones here?

Reply:
I second this question

Reply:
There are three sanitisers that we can use: address, undefined behaviour and thread. (There is also a memory one but I've not got that working yet). The thread and address sanitisers are mutually exclusive in their application so we need a separate program to allow us to run the thread sanitiser (the main tests run with the address sanitiser). The reason that we include the thread tests here is because we are just trying to catch errors (like memory frees, etc) encountered by the use in threads while the thread test binary catches data races.

Long story short, we run them in both to allow all the sanitisers to catch problems even though we don't necessarily assert.